### ISSUE TITLE
The check of `block-opening-brace-newline-after` says nothing about the run PostCSS prints in front of a node carrying no `raws.before`

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#680]
### What minimal example or steps are needed to reproduce the bug?

`block-opening-brace-newline-after` reads the run standing behind a block's opening brace out of the `raws.before` of the node the run stands in front of, through `rawNodeString`, which reads a raw that is not a string as the empty run. A node carrying no such raw therefore answers `never-multi-line` with no whitespace behind the brace, and no warning is raised — while PostCSS prints a run of its own in front of that node, one holding a line break.

So a `--fix` run writes the very break the option refuses, and says nothing about it. The shape is out of reach of the plugin alone: every child of a parsed block carries a run of its own, and nothing the plugin ships builds a node or moves one. A rule of another plugin, inserting a node at the head of a block in the same run, reaches it — this one fills an empty block with two declarations:

```js
let foreign = stylelint.createPlugin(`foreign/fill-empty-block`, () => (root) => {
	root.walkRules((statement) => {
		if (statement.nodes.length !== 0) return

		let first = new Declaration({ prop: `color`, value: `pink` })
		let second = new Declaration({ prop: `top`, value: `0` })

		second.raws.before = `\n\t`
		statement.append(first)
		statement.append(second)
	})
})
```

Over `a {\n}\n`, `--fix` gives `a {\n\tcolor: pink;\n\ttop: 0\n}\n` and raises no warning at all.

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/block-opening-brace-newline-after": "never-multi-line"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### What did you expect to happen?

The run PostCSS prints in front of such a node is read as the run behind the brace, so `never-multi-line` reports it and the fix takes it out.

### What actually happened?

Nothing is reported, and `--fix` leaves a block broken across lines with whitespace behind its opening brace.

### Do you have a proposal to fix the bug?

This is the reader's half of [#411](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/411), which fixed the writer's: that branch settled that a run PostCSS invents in front of a node carrying no raw is the rule's business, and writes the empty run for every node of the comment run the fix walks. The check has not been asked the same question. Whichever way it is answered, the two halves want one answer: either a missing raw is the empty run for both, and the file that comes out of `--fix` still holds a break nobody reported, or it is the run PostCSS will print, and both the check and the fix read it that way.

Found while working on #411; it reproduces on `4fe5baa`, the commit that branch stands on, exactly as it does after it.
